src/Components/CreateImageWizardV2/steps/Packages/Packages.tsx

```diff
@@ -760,91 +952,202 @@ const Packages = () => {
-              onSelect: (event, isSelecting) =>
-                handleSelect(pkg, rowIndex, isSelecting),
-            }}
-          />
-          <Td>{pkg.name}</Td>
-          <Td>
-            {pkg.summary ? (
-              pkg.summary
+    let rows: ReactElement[] = [];
+    rows = rows.concat(
+      transformedGroups
+        .slice(computeStart(), computeEnd())
+        .map((grp, rowIndex) => (
+          <Tr key={`${grp.name}-${rowIndex}`} data-testid="package-row">
+            <Td
+              select={{
+                isSelected: groups.some((g) => g.name === grp.name),
+                rowIndex: rowIndex,
+                onSelect: (event, isSelecting) =>
+                  handleGroupSelect(grp, rowIndex, isSelecting),
+              }}
+            />
+            <Td>
+              @{grp.name}
+              <Popover
+                bodyContent={
+                  <div style={{ height: '40em', overflow: 'scroll' }}>
+                    <Table variant="compact">
+                      <Thead>
+                        <Tr>
+                          <Th>Included packages</Th>
+                        </Tr>
+                      </Thead>
+                      <Tbody>
+                        {grp.package_list.map((pkg) => (
+                          <Tr key={`details-${pkg}`}>
+                            <Td>{pkg}</Td>
+                          </Tr>
+                        ))}
+                      </Tbody>
+                    </Table>
+                  </div>
+                }
+              >
+                <Button
+                  variant="plain"
+                  aria-label="About included repositories"
+                  component="span"
+                  className="pf-u-p-0"
+                  isInline
+                >
+                  <HelpIcon className="pf-u-ml-xs" />
+                </Button>
+              </Popover>
+            </Td>
+            <Td>
+              {grp.description ? (
+                grp.description
+              ) : (
+                <span className="not-available">Not available</span>
+              )}
+            </Td>
+            {grp.repository === 'distro' ? (
+              <>
+                <Td>
+                  <img
+                    src={
+                      '/apps/frontend-assets/red-hat-logos/logo_hat-only.svg'
+                    }
+                    alt="Red Hat logo"
+                    height={RH_ICON_SIZE}
+                    width={RH_ICON_SIZE}
+                  />{' '}
+                  Red Hat repository
+                </Td>
+                <Td>Supported</Td>
+              </>
+            ) : grp.repository === 'custom' ? (
+              <>
+                <Td>Third party repository</Td>
+                <Td>Not supported</Td>
+              </>
             ) : (
-              <span className="not-available">Not available</span>
+              <>
+                <Td className="not-available">Not available</Td>
```

Comment:
"Recommended" groups seem to be defaulting to "Not available" for repo and support

![image](https://github.com/osbuild/image-builder-frontend/assets/49452678/6ab5026d-390a-4aff-8d2b-dff8e41da1e2)

Should those default to "EPEL" and not supported as it's the only recommendation option currently?